src/vs/workbench/workbench.common.main.ts

```diff
@@ -518,6 +518,10 @@ import 'sql/workbench/contrib/resourceDeployment/browser/resourceDeployment.cont
 // Extension
 import 'sql/workbench/contrib/extensions/browser/extensions.contribution';
 
+// TSGOps ads light image
+// eslint-disable-next-line code-import-patterns
```

Comment:
You should almost never be disabling eslint rules - and specifically you should never disable this one. Do you actually need to disable this? It should be fine...

Reply:
It was giving me some warning. I will try to remove it and see